tests/unit/dataactbroker/test_account_handler.py

```diff
@@ -0,0 +1,103 @@
+from unittest.mock import Mock
+import dataactbroker.handlers.accountHandler
+import json
+from dataactcore.utils.jsonResponse import JsonResponse
+from dataactcore.utils.statusCode import StatusCode
+
+
+def test_max_login_success(database, user_constants, monkeypatch):
+    ah = dataactbroker.handlers.accountHandler.AccountHandler(Mock())
+
+    mock_dict = Mock()
+    mock_dict.return_value.safeDictionary.side_effect = {'ticket': '', 'service': ''}
+    monkeypatch.setattr(dataactbroker.handlers.accountHandler, 'RequestDictionary', mock_dict)
+
+    max_dict= {'cas:serviceResponse': {}}
+    monkeypatch.setattr(ah, 'get_max_dict', Mock(return_value=max_dict))
+    config = {'parent_group': 'parent-group'}
+    monkeypatch.setattr(dataactbroker.handlers.accountHandler, 'CONFIG_BROKER', config)
+    max_dict = {
```

Comment:
That may be overkill as this is the only test that needs it. I would opt for a fixture if other tests could also use it.